### PR TITLE
Add VirtualMachineCRCErrors runbook

### PR DESCRIPTION
**Pull request by machadovilaca**
Runbook for alert in PR https://github.com/kubevirt/kubevirt/pull/9741

### CODE REVIEW

File: docs/runbooks/VirtualMachineCRCErrors.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+$ kubectl get sc <storage-class-name> -o yaml
+```
+
+4) Check if the storage class has the "krbd:rxbounce" map option:
+
+## Mitigation
+
+Add the "krbd:rxbounce" map option to your storage class configuration.
+
+<!--DS: Check https://access.redhat.com/articles/6978371 for more information.-->
```
Comment (sradco) [inline docs/runbooks/VirtualMachineCRCErrors.md:41]: No need for the d/s text here.

File: docs/runbooks/VirtualMachineCRCErrors.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+```bash
+$ kubectl get sc <storage-class-name> -o yaml
+```
+
+4) Check if the storage class has the "krbd:rxbounce" map option:
+
+## Mitigation
+
+Add the "krbd:rxbounce" map option to your storage class configuration.
```
Comment (sradco) [inline docs/runbooks/VirtualMachineCRCErrors.md:39]: Can we add an example?

Comment (sradco) [reply docs/runbooks/VirtualMachineCRCErrors.md:39]: Can you please also share what is the "krbd:rxbounce" map ?

**Comment by apinnick** [comment]
Is this runbook for 4.14.0?

File: docs/runbooks/VirtualMachineCRCErrors.md (hunk 1 above)
Comment (machadovilaca) [reply docs/runbooks/VirtualMachineCRCErrors.md]: removed

File: docs/runbooks/VirtualMachineCRCErrors.md (hunk 2 above)
Comment (machadovilaca) [reply docs/runbooks/VirtualMachineCRCErrors.md]: added

**Comment by machadovilaca** [comment]
@apinnick

> Is this runbook for 4.14.0?

yes

**Comment by apinnick** [comment]
Is there a downstream epic for this feature? I will have to create some doc tasks.

**Review by sradco — APPROVED** [review]

**Pull request merged**